services/app/src/components/base/Dialogs.tsx

```diff
@@ -208,6 +208,35 @@ export class ExpansionSelectDialog extends React.Component<ExpansionSelectDialog
   }
 }
 
+export class TooManyPlayersDialog<T extends BaseDialogProps> extends React.Component<T, {}> {
+  protected title: string;
+  protected content: JSX.Element;
+
+  constructor(props: T) {
+    super(props);
+    this.title = 'Too many players!';
+    this.content = <p>Expedition can only be played by a maximum of 6 players!</p>;
```

Comment:
I'd replace the exclamation mark in contents with a period, since there's already one in the title and too many exclamation marks comes off a bit strong

Reply:
Alrighty. Thanks.